include/iso_alloc_ds.h

```diff
@@ -152,6 +152,9 @@ typedef struct {
     int32_t big_zone_free_count;
     int32_t big_zone_used_count;
     uint16_t zones_used;
+#if ARM_MTE
+    bool arm_mte_enabled;
```

Comment:
This should go in a read-only zone I think, otherwise, MTE is disableable by an attacker with an arbitrary write primitive.

Reply:
A read-only section for settings like this is easy to implement but often requires dereferencing both `_root` and `ro_config_page` which can flush the cache and really hurt performance in any hot paths.